internal/accrual/accrual.go

```diff
@@ -0,0 +1,128 @@
+package accrual
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/fortuna91/ya_praktikum_final/internal/db"
+	"github.com/fortuna91/ya_praktikum_final/internal/utils"
+	"log"
+	"net/http"
+	"strconv"
+	"sync"
+	"time"
+)
+
+const (
+	REGISTERED string = "REGISTERED"
+	PROCESSING string = "PROCESSING"
+	// INVALID           = "INVALID"
+	// PROCESSED         = "PROCESSED"
+)
+
+type QueueAccrualSystem struct {
+	Queue                []db.OrderData
+	AccrualSystemAddress string
+	RetryAfter           int
+	mtx                  sync.RWMutex
+}
+
+func sendRequest(client *http.Client, request *http.Request) *http.Response {
+	request.Header.Set("Content-Length", "0")
+	response, err := client.Do(request)
+	if err != nil {
+		fmt.Println(err)
+		return nil
+	}
+	return response
+}
+
+func getAccrual(accrualSystemAddress string, orderID string) (*db.OrderData, int) {
+	client := http.Client{}
+	request, _ := http.NewRequest(http.MethodGet, accrualSystemAddress+"/api/orders/"+orderID, nil)
+	response := sendRequest(&client, request)
+	if response == nil {
+		log.Println("Error in getting accrual")
+		return nil, 0
+	}
+	if response.StatusCode == 429 {
```

Comment:
```suggestion
	if response.StatusCode == http.StatusTooManyRequests {
```
предлагаю пользоваться стандартными константами